plotly/tests/test_optional/test_offline/test_offline.py

```diff
@@ -0,0 +1,31 @@
+"""
+test__offline
+
+"""
+from __future__ import absolute_import
+from nose.tools import raises
+import os
+
+
+from plotly.exceptions import PlotlyError
+import plotly
+
+
+@raises(PlotlyError)
+def test_calling_iplot_before_initializing_raises_an_error():
+    plotly.offline.iplot([{'x': [1, 2, 3]}])
+
+
+def test_no_errors_are_raised_when_properly_initializing_offline_mode():
+    plotly.offline.init_notebook_mode()
+    plotly.offline.iplot([{'x': [1, 2, 3]}])
```

Comment:
returns `None`. We could refactor `iplot` to call some private functions that we could test..